lsp/core/aws-lsp-core/src/content/cache/uriCacheRepository.ts

```diff
@@ -0,0 +1,159 @@
+import { createHash } from 'crypto'
+import * as fs from 'fs'
```

Comment:
Will these 'fs' imports need to be revisited for supporting the browser? Or will there be a different impl of uriCacheRepository for browser?

Reply:
TBD - these might get patched for browser use, or likely an alternate impl will be used. This sort of thing will likely become the responsibility of the "runtime" component, handled by a separate team. Our current focus is not on JSON Schemas, so we won't be working in this space for a while.